00-Basics/03-SmallestFromFour/min.cpp

```diff
@@ -0,0 +1,36 @@
+#include <iostream>
+using namespace std;
+
+int min_of_four( int a,int b,int c, int d)
+{
+	int min = a;
+	
+     if (min>b)
+	 {
+  	    min=b;
+     }
+     else
+	 {
+	       if (min>c) 
+	       {
+        	min=c;	
+           }	
+           else if (min>d)
+	       {
+  	        min=d;
+           }
+	 }
+    return min;
+}
```

Comment:
Here the logic is bad, test it with input `10,20,1,2` your output will be 10 instead of  1. 
And again, if you indent properly, it will help to see, why the code is wrong.

Reply:
@hegyhati here !

Reply:
Sorry, I gave bad input sequence. The code doesn't work for `20,10,1,2` properly